GiniCapture/Classes/Core/Screens/Help/HelpMenuViewController.swift

```diff
@@ -106,10 +79,46 @@ final public class HelpMenuViewController: UITableViewController {
         }
     }
     
+    fileprivate func configureMainView() {
+        title = .localized(resource: HelpStrings.menuTitle)
+    }
+    
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+        configureMenuItems()
+        configureMainView()
+        configureTableView()
+    }
+
+    func configureMenuItems() {
+        let title: String = .localized(resource: ImageAnalysisNoResultsStrings.titleText)
+        let topViewText: String = .localized(resource: ImageAnalysisNoResultsStrings.warningHelpMenuText)
+        let viewController = ImageAnalysisNoResultsViewController(title: title,
+                                                                  subHeaderText: nil,
+                                                                  topViewText: topViewText,
+                                                                  topViewIcon: nil)
+        let noResultsItemTitle: String = .localized(resource: HelpStrings.menuFirstItemText)
+        let noResultsItem = MenuItem(title: noResultsItemTitle, viewController: viewController, presentationMode: ItemPresentationMode.noResultsTips)
+        menuItems.append(noResultsItem)
+
+        if giniConfiguration.shouldShowSupportedFormatsScreen {
+            let supportedFormatsItem = MenuItem(title: .localized(resource: HelpStrings.menuThirdItemText), viewController: SupportedFormatsViewController())
+            menuItems.append(supportedFormatsItem)
+        }
+
+        if giniConfiguration.openWithEnabled {
+            let openWithItem = MenuItem(title: .localized(resource: HelpStrings.menuSecondItemText), viewController: OpenWithTutorialViewController())
+            menuItems.append(openWithItem)
+        }
+        
+        if giniConfiguration.hasCustomMenuItems {
+            menuItems.append(contentsOf: giniConfiguration.customMenuItems)
+        }
+    }
```

Comment:
Then in this method we simply add the default and custom menu items to `menuItems`:
```suggestion
    func configureMenuItems() {
        menuItems.append(contentsOf: Item.defaultItems())
        menuItems.append(contentsOf: giniConfiguration.customMenuItems)
    }
```

Continued below ->